xcdat/temporal.py

```diff
@@ -400,6 +408,9 @@ def climatology(
 
             The weight of masked (missing) data is excluded when averages are
             taken. This is the same as giving them a weight of 0.
+
+            Warning: If one time point spans across the time intervals that 
+            you are averaging into, then weights are not properly assigned.
```

Comment:
Here and elsewhere – should this be a longer entry at the bottom of the docstring (like the example above). Maybe we could also include a brief note, like:

"Note that weights are assigned by the labelled time point. If the dataset includes timepoints that span across typical boundaries (e.g., a timepoint on `2020-06-01` with bounds that begin in May 2020 and end in June 2020), the weights will not be assigned properly. See explanation below."